doc/classes/String.xml

```diff
@@ -91,7 +91,7 @@
 		<method name="capitalize" qualifiers="const">
 			<return type="String" />
 			<description>
-				Changes the appearance of the string: replaces underscores ([code]_[/code]) with spaces, adds spaces before uppercase letters in the middle of a word, converts all letters to lowercase, then converts the first one and each one following a space to uppercase.
+				Changes the appearance of the string: replaces underscores ([code]_[/code]) with spaces, adds spaces before uppercase letters in the middle of a word, converts all letters to lowercase, then converts the first letter and each letter following a space to uppercase.
```

Comment:
If we're changing this for clarity, I think there's more work needed to make it actually clear.

```suggestion
				Returns a copy of the string with each word capitalized, i.e. starting with an uppercase letter. The following conversions are applied:
				- Underscore characters ([code]_[/code]) are replaced with spaces (i.e. converts [code]snake_case[/code] to natural language).
				- Words with uppercase characters in the middle of the word are split into separate words (i.e. converts [code]camelCase[/code] to natural language).
				- After applying the above rules to create multiple words, the first letter of each word is made uppercase. The remaining letters of each word are made lowercase.
```